client/jetpack-cloud/sections/partner-portal/billing-details/index.tsx

```diff
@@ -0,0 +1,163 @@
+						total: 1258,
+					},
+					productTotalCost: 174862,
+				},
+			],
+		},
+	};
+
+	return (
+		<div className="billing-details">
+			<Card compact className="billing-details__header">
+				<div className="billing-details__row">
+					<div>{ translate( 'Products' ) }</div>
+					<div>{ translate( 'Assigned' ) }</div>
+					<div>{ translate( 'Unassigned' ) }</div>
+					<div></div>
+				</div>
+			</Card>
+
+			{ billing.isSuccess &&
+				billing.data.products.map( ( product ) => (
+					<Card compact key={ product.productSlug }>
+						<div className="billing-details__row">
+							<div className="billing-details__product">
+								{ product.productName }
+								<span className="billing-details__line-item-meta">
+									{ translate( 'Price per license: %(price)s', {
+										args: { price: formatCurrency( product.productCost, 'USD' ) },
+									} ) }
+								</span>
+							</div>
+
+							<div className="billing-details__assigned">
+								{ product.counts.assigned }
+								<span className="billing-details__line-item-meta billing-details__line-item-meta--is-mobile">
+									{ translate( 'Assigned' ) }
+								</span>
+							</div>
+
+							<div className="billing-details__unassigned">
+								{ product.counts.unassigned }
+								<span className="billing-details__line-item-meta billing-details__line-item-meta--is-mobile">
+									{ translate( 'Unassigned' ) }
+								</span>
+							</div>
+
+							<div className="billing-details__subtotal">
+								{ translate( '%(count)d License', '%(count)d Licenses', {
+									count: product.counts.total,
+									args: { count: product.counts.total },
+								} ) }
+								<span className="billing-details__line-item-meta">
+									{ translate( 'Subtotal: %(subtotal)s', {
+										args: { subtotal: formatCurrency( product.productTotalCost, 'USD' ) },
+									} ) }
+								</span>
+							</div>
+						</div>
+					</Card>
+				) ) }
+
+			<Card compact className="billing-details__footer">
+				<div className="billing-details__row billing-details__row--summary">
+					<span className="billing-details__total-label billing-details__cost-label">
+						{ billing.isSuccess &&
+							translate( 'Cost for {{bold}}%(date)s{{/bold}}', {
+								components: { bold: <strong /> },
+								args: { date: moment( billing.data.date ).format( 'MMMM, YYYY' ) },
+							} ) }
+					</span>
+					<strong className="billing-details__cost-amount">
+						{ billing.isSuccess && formatCurrency( billing.data.costs.total, 'USD' ) }
+					</strong>
+
+					<span className="billing-details__total-label billing-details__line-item-meta">
+						{ translate( 'Assigned licenses:' ) }
+					</span>
+					<span className="billing-details__line-item-meta">
+						{ billing.isSuccess && formatCurrency( billing.data.costs.assigned, 'USD' ) }
+					</span>
+
+					<span className="billing-details__total-label billing-details__line-item-meta">
+						{ translate( 'Unassigned licenses:' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'No unassigned licenses.' )` *ES Score: 12*